scripts/core/editor3/components/Editor3.jsx

```diff
@@ -20,146 +21,137 @@ import Toolbar from './toolbar';
  * @description Editor3 is a draft.js based editor that support customizable
  *  formatting (shortly more features will come ...)
  */
-export class Editor3 extends React.Component {
-    constructor(props) {
-        super(props);
 
-        const initialContentState = stateFromHTML(props.value);
+export const getInitialState = (spellcheckerService, value, onChangeValue, readOnly, showToolbar, editorFormat) => {
```

Comment:
Yes, usually the store is at the application level, here we use it just for Editor3. I think we can refactor the creation of the store when we extend the using of react&redux at edit content item page.